code/game/gamemodes/cult/runes.dm

```diff
@@ -196,6 +196,11 @@
 			M << "<span class='sinister'>You can now speak and understand the forgotten tongue of the occult.</span>"
 			M.add_language("Cult")
 			log_admin("[usr]([ckey(usr.key)]) has converted [M] ([ckey(M.key)]) to the cult at [M.loc.x], [M.loc.y], [M.loc.z]")
+			if(M.client)
+				spawn(100)//10 seconds sounds good
```

Comment:
wouldn't sleep cause issues if the rune was erased before the 10 seconds are up?